rb_ws/src/buggy/scripts/auton/autonsystem.py

```diff
@@ -125,11 +124,13 @@ def init_check(self):
             return False
 
         # waits until covariance is acceptable to check heading
-
         with self.lock:
             self_pose, _ = self.get_world_pose_and_speed(self.self_odom_msg)
             current_heading = self_pose.theta
             closest_heading = self.cur_traj.get_heading_by_index(trajectory.get_closest_index_on_path(self_pose.x, self_pose.y))
+        print("current heading: ", np.rad2deg(current_heading))
```

Comment:
Does this clog up std:out?

Reply:
that is the point, it pauses here until we can proceed.